src/main/scala/com/twitter/penguin/korean/tokenizer/KoreanTokenizer.scala

```diff
@@ -41,26 +41,41 @@ object KoreanTokenizer {
   private val TOP_N_PER_STATE = 5
   private val MAX_TRACE_BACK = 8
 
-  // Lower score is better
-  private val WEIGHT_TOKENS = 0.18f
-  private val WEIGHT_UNKNOWNS = 0.3f
-  private val WEIGHT_WORDS = 0.3f
-  private val WEIGHT_UNKNOWN_COVERAGE = 0.5f
-  private val WEIGHT_FREQ = 0.2f
-  private val WEIGHT_POS_UNKNOWNS = 10.0f
-  private val WEIGHT_EXACT_MATCH = 0.5f
-  private val WEIGHT_ALL_NOUN = 0.1f
-  private val WEIGHT_PREFFERED_PATTERN = 0.6f
-  private val WEIGHT_DETERMINER = -0.01f
-  private val WEIGHT_EXCLAMATION = 0.01f
-
-  // supress suffix when tied
-  private val WEIGHT_INITIAL_POSTPOSITION = 0.2f
-
-  private val WEIGHT_HA_VERB = 0.3f
-
-  private val PREFERRED_PATTERNS = Seq(Seq(Noun, Josa), Seq(ProperNoun, Josa))
+  case class KoreanTokenizerParameters(
+    // Lower score is better
+    WEIGHT_TOKENS: Float = 0.18f,
+    WEIGHT_UNKNOWNS: Float = 0.3f,
+    WEIGHT_WORDS: Float = 0.3f,
+    WEIGHT_UNKNOWN_COVERAGE: Float = 0.5f,
+    WEIGHT_FREQ: Float = 0.2f,
+    WEIGHT_POS_UNKNOWNS: Float = 10.0f,
+    WEIGHT_EXACT_MATCH: Float = 0.5f,
+    WEIGHT_ALL_NOUN: Float = 0.1f,
+    WEIGHT_PREFFERED_PATTERN: Float = 0.6f,
+    WEIGHT_DETERMINER: Float = -0.01f,
```

Comment:
Thanks for the change! Please switch to lower-camel cases as they are not constants any more.